schemas/2015-11-01/Microsoft.Network.json

```diff
@@ -0,0 +1,231 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2015-11-01/Microsoft.Network.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Network",
+  "description": "Microsoft Network Resource Types",
+  "resourceDefinitions": {
+    "trafficManagerProfiles": {
+      "type": "object",
+      "description" : "An Azure Traffic Manager profile",
+      "properties": {
+        "apiVersion": {
+          "enum": [
+            "2015-11-01"
+          ]
+        },
+        "type": {
+          "enum": [
+            "Microsoft.Network/trafficManagerProfiles"
+          ]
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "profileStatus": {
+              "anyOf": [
```

Comment:
Vlad, please still use "oneOf", we need exact one match here, instead of "at least one". The error may be an issue of our test tool, I will work with @daschult on it.